src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -95,8 +96,8 @@ public void simulationPeriodic() {
 		topFlywheelSim.update(0.020);
 		bottomFlywheelSim.update(0.020);
 
-		topMotorSpeedSim += topFlywheelSim.getAngularVelocityRPM();
-		bottomMotorSpeedSim += bottomFlywheelSim.getAngularVelocityRPM();
+		topMotorSpeedSim = topFlywheelSim.getAngularVelocityRPM();
+		bottomMotorSpeedSim = bottomFlywheelSim.getAngularVelocityRPM();
```

Comment:
Does the simulator work now?

Reply:
It is showing signs of life. The isSimulation() test was backwards.